CODEOWNERS

```
@@ -412,6 +412,7 @@ Makefile* @cilium/build
 /pkg/datapath/linux/bandwidth/ @cilium/sig-datapath
 /pkg/bgp/ @cilium/sig-bgp
 /pkg/bgpv1/ @cilium/sig-bgp
+/pkg/bitlpm/ @cilium/ipcache @cilium/sig-policy
```

Codeowners check is failing.
```suggestion
/pkg/container/bitlpm/ @cilium/ipcache @cilium/sig-policy
```